fire/fire_import_test.py

```diff
@@ -12,16 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import fire
-
+import mock
+import sys
 import unittest
 
+import fire
+
 
 class FireImportTest(unittest.TestCase):
   """Tests importing Fire."""
 
   def testFire(self):
-    fire.Fire()
+    with mock.patch.object(sys, 'argv', ['commandname']):
+        fire.Fire()
```

Comment:
We have something like a pylintrc that I want to get checked into this project. The problem is it has a number of things that are internal specific, so it'll take some cleanup before I'm able to add it here. Definitely would like to get it into travis soon though.

I think it's pretty much this https://google.github.io/styleguide/pyguide.html, but I know there are some differences (e.g. FunctionNames are permitted.)